### PR TITLE
feat: Add internationalization support

### LINKED ISSUE #729: How to translate the language of the parse-dashboard ?
How to translate the language of the parse-dashboard ? I want to translate the language to chinaese

### PR DESCRIPTION
**Pull request by badboy-tian**
### New Pull Request Checklist

- [ ] I am not disclosing a [vulnerability](https://github.com/parse-community/parse-server/blob/master/SECURITY.md).
- [x] I am creating this PR in reference to an [issue](https://github.com/parse-community/parse-dashboard/issues/729).

### Issue Description

add multi language support, simple demo
Closes: #729 

### TODOs before merging

- [ ] Add tests
- [ ] Add changes to documentation (guides, repository pages, in-code descriptions)

### CODE REVIEW

File: src/dashboard/Dashboard.js [hunk 1]
```diff
@@ -193,9 +195,12 @@ export default class Dashboard extends React.Component {
     }
 
     const AppsIndexPage = () => (
-      <AccountView section='Your Apps'>
-        <AppsIndex newFeaturesInLatestVersion={this.state.newFeaturesInLatestVersion}/>
-      </AccountView>
+      <Translation>
+        {t =>
+          <AccountView section={t('Your Apps')}>
```
Comment (mtrezza) [inline src/dashboard/Dashboard.js]: This looks overly complex, just to translate the string `Your Apps`. Is there a less intrusive way?

File: src/dashboard/Dashboard.js [hunk 2]
```diff
@@ -5,54 +5,56 @@
...
-import Webhooks           from './Data/Webhooks/Webhooks.react';
-import { AsyncStatus }    from 'lib/Constants';
-import baseStyles         from 'stylesheets/base.scss';
-import { get }            from 'lib/AJAX';
-import { setBasePath }    from 'lib/AJAX';
+import PushDetails from './Push/PushDetails.react';
+import PushIndex from './Push/PushIndex.react';
+import PushNew from './Push/PushNew.react';
+import PushSettings from './Settings/PushSettings.react';
+import React from 'react';
```
Comment (mtrezza) [inline src/dashboard/Dashboard.js]: There seems to be a style issue, please revert these changes / run the linter.

**Review by mtrezza — CHANGES_REQUESTED** [review]

File: src/dashboard/Dashboard.js (hunk 1 above)
Comment (badboy-tian) [reply src/dashboard/Dashboard.js]: ”Your Apps“ is the key
zh.json
"Your Apps": "APP列表"
en.json
"Your Apps": "Your Apps"

Comment (mtrezza) [reply src/dashboard/Dashboard.js]: Yes, but why is the whole translation tag needed, if we are only changing the string. Is there a shorter version?

Comment (badboy-tian) [reply src/dashboard/Dashboard.js]: https://react.i18next.com/latest/translation-render-prop 
With the trans component， should wrap the <Translation> tag to whole translation tag

Comment (mtrezza) [reply src/dashboard/Dashboard.js]: Are there alternative modules for i18n? Did you compare other options, what are the benefits of this one?

Comment (badboy-tian) [reply src/dashboard/Dashboard.js]: Sorry, I have no idea about 'alternative modules'. https://github.com/i18next/react-i18next has 8k+ 'Star'. This program has so many 'Star', so it should be very popular and have its advantages

Comment (mtrezza) [reply src/dashboard/Dashboard.js]: Looking at the playground code of that package, the implementation looks much simpler:

```
const { t, i18n } = useTranslation();
...
return (
   <h2>{t("Welcome to React")}</h2>
)
```

If I'm not mistaken, they use no ` <Translation> {t =>` tags. Could you refactor to achieve this simplicity?

File: package.json [hunk 3]
```diff
@@ -66,12 +68,14 @@
...
     "react-helmet": "6.1.0",
+    "react-i18next": "^12.3.1",
     "react-json-view": "1.21.3",
     "react-popper-tooltip": "4.4.2",
     "react-router-dom": "6.4.1",
     "regenerator-runtime": "0.13.11",
     "semver": "7.3.7",
-    "typescript": "4.8.3"
+    "typescript": "4.8.3",
+    "uuid": "^9.0.0"
```
Comment (mtrezza) [inline package.json]: Please remove the carets, we use fixed versions.

**Review by mtrezza — CHANGES_REQUESTED** [review]

**Review by mtrezza — CHANGES_REQUESTED** [review]
The package-lock file contains a lot of changes; and the lock file version changes from 1 to 3. Could you please completely revert the lock file and then just run (preferably on Node 16):

```
npm i -E i18next i18next-browser-languagedetector react-i18next uuid
```

I couldn't find where UUID is used, please also verify that all the packages above are really required in this PR. Please also take a look at the tests, for example the failing lint.

**Comment by badboy-tian** [comment]
> The package-lock file contains a lot of changes; and the lock file version changes from 1 to 3. Could you please completely revert the lock file and then just run (preferably on Node 16):
> 
> ```
> npm i -E i18next i18next-browser-languagedetector react-i18next uuid
> ```
> 
> I couldn't find where UUID is used, please also verify that all the packages above are really required in this PR. Please also take a look at the tests, for example the failing lint.

```
ERROR in ../node_modules/parse/lib/browser/uuid.js 4:7-25
Module not found: Error: Can't resolve 'uuid/v4' in 'D:\web\parse-dashboard\node_modules\parse\lib\browser'
```

**Comment by mtrezza** [comment]
Why is the package suddenly needed? It wasn't needed before?

File: src/dashboard/Dashboard.js (hunk 1 above)
Comment (badboy-tian) [reply src/dashboard/Dashboard.js]: ok, got it

File: package.json [hunk 4]
```diff
@@ -112,7 +115,8 @@
     "svg-prep": "1.0.4",
     "webpack": "5.75.0",
     "webpack-cli": "5.0.1",
-    "yaml": "1.10.0"
+    "yaml": "1.10.0",
+    "uuid": "9.0"
```
Comment (mtrezza) [inline package.json]: This needs to be a semver string, e.g. `9.0.0`.

**Review by mtrezza — CHANGES_REQUESTED** [review]

**Comment by mtrezza** [comment]
Ci is still failing, could you take a look? Btw, you can run the tests locally to find out whether there are any issues, so you don't have to wait for the CI.
